pkg/distsql/request_builder.go

```diff
@@ -879,3 +885,11 @@ func appendRanges(tbl *model.TableInfo, tblID int64) ([]kv.KeyRange, error) {
 	}
 	return retRanges, nil
 }
+
+// FulltextIndexRangesToKVRanges converts fulltext index ranges to "KeyRange".
+// This is a temporary solution, and we should implement the fulltext index
+// ranges conversion in the future.
+func FulltextIndexRangesToKVRanges(dctx *distsqlctx.DistSQLContext, tids []int64, ranges []*ranger.Range) (*kv.KeyRanges, error) {
+	// Mocking the fulltext index ranges to be the same as the table ranges.
+	return CommonHandleRangesToKVRanges(dctx, tids, ranges)
```

Comment:
Should it be `TableRangesToKVRanges` instead?

Reply:
Now this pr is just a mock.  Return `CommonHandleRangesToKVRanges` or `TableRangesToKVRanges` both are ok..